src/components/TransactionList/TransactionList.jsx

```diff
@@ -0,0 +1,68 @@
+import { DeleteIcon, EditIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Flex,
+  HStack,
+  IconButton,
+  Tooltip,
+  VStack
+} from '@chakra-ui/react';
+import { format } from 'date-fns';
+
+import { NoteIcon } from '@/assets';
+import { useCentralTheme } from '@/theme';
+
+export const TransactionList = ({ list }) => {
```

Comment:
Maybe we also need to get values about what kind of list we should do? I mean, accept an prop like 'Income' or 'Expense' or 'all' of them